app/jobs/decidim/action_delegator/send_sms_job.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require "savon"
+
+module Decidim
+  module ActionDelegator
+    class SendSmsJobException < StandardError; end
+    class SendSmsJob < ApplicationJob
+      queue_as :default
+
+      attr_reader :response
+
+      SMSVIRTUAL_WSDL_URL = "https://websms.masmovil.com/api_php/smsvirtual.wsdl"
```

Comment:
I would call freeze as well so we ensure it's really a constant

Reply:
rubocop tells me to not freeze immutable objects :)

Reply:
aah right